docs/modules/apa102.md

```diff
@@ -9,6 +9,10 @@ This module provides Lua access to [APA102 RGB LEDs](https://youtu.be/UYvC-hukz-
 
 source: [Adafruit](https://www.adafruit.com/products/2343)
 
+!!! caution
+
+    This module will work without the [pixbuf module](pixbuf.md). The respective parts will just not be available.
```

Comment:
Can I propose an alternative (for all except the effects lib).

> This module has an _optional_ dependency to the [pixbuf module](pixbuf.md) i.e. it can work without. However, if you compile the firmware without pixbuf the respective features will be missing from this module.

Reply:
Changed it